### PR TITLE
Add csh fish activation scripts

### LINKED ISSUE #30: activation scripts needed for csh/tcsh
This is mostly a note to myself to add activation scripts for other shells.  See https://github.com/conda-forge/nco-feedstock/issues/103, where the lack thereof caused problems.

### PR DESCRIPTION
**Pull request by xylar**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

closes #30

### CODE REVIEW

**Comment by xylar** [comment]
@conda-forge-admin, please rerende

File: recipe/scripts/deactivate.csh [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env csh
+
+# Restore previous env vars if they were set.
+unsetenv UDUNITS2_XML_PATH
+
+if ( $?_CONDA_SET_UDUNITS2_XML_PATH ) then
+    setenv UDUNITS2_XML_PATH "$_CONDA_SET_UDUNITS2_XML_PATH"
+    unsetenv _CONDA_SET_UDUNITS2_XML_PATH
+endif
```
Comment (xylar) [inline recipe/scripts/deactivate.csh:9]:
```suggestion
endif

```

**Comment by xylar** [comment]
@conda-forge-admin, please rerender

**Comment by zklaus** [comment]
Thanks for this! Looks mostly good to me, but could you explain why the bld.bat is not needed?

**Comment by xylar** [comment]
@zklaus, the `build.bat` in `recipe` is fine and I left it alone.  I believe the `build.bat` in `recipe/scripts` was probably added by mistake.  But I'm happy to track down the commit and make sure.

**Comment by xylar** [comment]
The extra copy at `recipe/scripts/build.bat` was added by @ocefpaf in https://github.com/conda-forge/udunits2-feedstock/pull/19 without any explanation.  @ocefpaf, could you confirm that that was probably just an accidental copy?

**Comment by xylar** [comment]
I forgot to add copying the scripts to the appropriate directories.  Switching this PR to draft until I fix that.

**Comment by zklaus** [comment]
Ah yes, silly me, I didn't pay enough attention to the path. Looks reasonable to me. @ocefpaf ?

**Comment by xylar** [comment]
In my testing, `csh` works but `fish` doesn't.  Trying to figure out why.

**Comment by xylar** [comment]
It turns out that my `fish` testing wasn't using the local package build.  Once I fixed that, `fish` is setting the expected environment variable, too.

**Comment by xylar** [comment]
@conda-forge/udunits2, this is ready for review.  Please let me know if you'd prefer I not add myself as a maintainer.  I don't feel strongly that I need to be, I just thought it was courteous to add myself if you could use the help.

**Comment by ocefpaf** [comment]
> Please let me know if you'd prefer I not add myself as a maintainer.

You are joking, right? I want you in all the feedstocks I maintain :stuck_out_tongue_winking_eye:

**Pull request merged**